seqr/views/utils/individual_utils.py

```diff
@@ -3,9 +3,13 @@
 """
 
 import logging
+from collections import defaultdict
 
-from seqr.models import Sample, IgvSample, Individual
+from seqr.models import Sample, IgvSample, Individual, Family
```

Comment:
is this used anywhere?

Reply:
The model `Family` is used in the function `add_or_update_individuals_and_families()` which was moved to here from `apis/individual_apis.py`.